src/components/cc-warning-payment/cc-warning-payment.js

```diff
@@ -35,21 +34,39 @@ export class CcWarningPayment extends LitElement {
   }
 
   render () {
+
+    const { title, error } = (this.mode === 'overview' || this.mode === 'billing') ? this._renderOrgaError(this.errors[0]) : '';
```

Comment:
Not really related to your modifications but isn't `_renderOrgaError` essentially a method to return i18n strings that we usually name `_get...` ?

It does not seem to return anything to render so I wonder if the name is not misleading.